module3/homework/inner_product/test.cpp

```diff
@@ -4,7 +4,7 @@
 #include "gtest/gtest.h"
 
 bool cmp(double first, double second, double epsilon = 0.5) {
-    return (fabs(first - second) < epsilon);
```

Comment:
It is possible to use fabs, you have to add one library, as I remember cmath or math.h